daemon/algod/api/server/v2/generated/routes.go

```diff
@@ -60,7 +60,9 @@ type ServerInterfaceWrapper struct {
 // AccountInformation converts echo context to params.
 func (w *ServerInterfaceWrapper) AccountInformation(ctx echo.Context) error {
 
-	validQueryParams := map[string]bool{}
+	validQueryParams := map[string]bool{
```

Comment:
given that this requires a memory allocation, you might want to refactor this to be a global map variable.

Reply:
Thanks for the reminder. Yes, this is an improvement that I'm aware of. It will require spending a bit more time ramping up on go's template engine.